pkg/host/internal/network/network.go

```diff
@@ -175,16 +179,42 @@ func (n *network) SetNetdevMTU(pciAddr string, mtu int) error {
 
 func (n *network) GetNetDevMac(ifaceName string) string {
 	log.Log.V(2).Info("GetNetDevMac(): get Mac", "device", ifaceName)
-	macFilePath := filepath.Join(vars.FilesystemRoot, consts.SysClassNet, ifaceName, "address")
-	data, err := os.ReadFile(macFilePath)
+	addressFilePath := filepath.Join(vars.FilesystemRoot, consts.SysClassNet, ifaceName, "address")
+	data, err := os.ReadFile(addressFilePath)
 	if err != nil {
-		log.Log.Error(err, "GetNetDevMac(): fail to read Mac file", "path", macFilePath)
+		log.Log.Error(err, "readInterfaceAddress(): fail to read address file", "path", addressFilePath)
```

Comment:
`readInterfaceAddress` is not the name of the functions

Reply:
True, will update that. Leftover while refactoring.